gcp/README.md

```diff
@@ -0,0 +1,100 @@
+# project_k8ssandra
+
+## Standerdization 
+
+terraform Style Guide to follow
+https://github.com/jonbrouse/terraform-style-guide/blob/master/README.md#file-names
+
+
+# folder Structure
+
+k8ssandra-terraform/
+  -aws/
+  -gcp/
+    -modules/
+      -vpc/
+        -main.tf 
+        -variables.tf 
+        -outputs.tf 
+        -README.md 
+      -iam/
+        -main.tf 
+        -variables.tf 
+        -outputs.tf 
+        -README.md
+      -gke/
+        -main.tf 
+        -variables.tf 
+        -outputs.tf 
+        -README.md
+    -README.md 
+    -env/
+      -dev.tf
+        ../modules/vpc
+        ../modules/iam
+        ../modules/gke_cluster
+      -version.tf 
+      -backend.tf 
+      -variables.tf 
+      -outputs.tf
+  -azure/
+  -tanzu/
+  -test
+  -scripts
+  -LICENSE
+  -Makefile
+  -README.md
+
+
+terraform version 0.14 
+Helm version v3.5.3
+
+### Google cloud
+Google Cloud SDK 333.0.0
+bq 2.0.65
+core 2021.03.19
+gsutil 4.60
+kubectl 1.17.17
+
+clone this repo, and to initialize the terraform run the following command
```

Comment:
Lets merge this code, but I need you to file an issue to replace all of this with a `make` command

Reply:
I have made the scripts and tested them locally. for the make command.